### PR TITLE
DROOLS-3908: [DMN Designer] Imported nodes should allow only outputs and not inputs

### PR DESCRIPTION
**Pull request by danielzhe**

### CODE REVIEW

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by jomarko** [comment]
Ok, seems code will change, I will wait. @danielzhe please let me know once ready for my review.

**Comment by danielzhe** [comment]
@manstis Thank you!!! I was wondering if it have some drawback of always returning `true` but I didn't found any until now. :D
Changes applied.

**Comment by manstis** [comment]
@jomarko This is ready for your review now.

**Comment by jomarko** [comment]
Looking right now!

File: kie-wb-common-dmn/kie-wb-common-dmn-api/src/test/java/org/kie/workbench/common/dmn/api/rules/NoInputNodesInImportedDecisionRuleTest.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+        assertTrue(actual);
+
+        verify(check).isReadOnly(any(Optional.class));
+    }
+
+    @Test
+    public void testAcceptWhenIsExpectedClass() {
+        when(rule.getId()).thenReturn(definition.getClass().getName());
+        boolean actual = check.accepts(rule, context);
```
Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-api/src/test/java/org/kie/workbench/common/dmn/api/rules/NoInputNodesInImportedDecisionRuleTest.java]: Just for consistency, please use `final boolean actual`

File: kie-wb-common-dmn/kie-wb-common-dmn-api/src/test/java/org/kie/workbench/common/dmn/api/rules/NoInputNodesInImportedDecisionRuleTest.java [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+        assertTrue(actual);
+    }
+
+    @Test
+    public void testAcceptWhenIsNotExpectedClass() {
+        when(rule.getId()).thenReturn("SomeOtherClass");
+        boolean actual = check.accepts(rule, context);
```
Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-api/src/test/java/org/kie/workbench/common/dmn/api/rules/NoInputNodesInImportedDecisionRuleTest.java]: Just for consistency, please use `final boolean actual`

File: kie-wb-common-dmn/kie-wb-common-dmn-api/src/main/java/org/kie/workbench/common/dmn/api/rules/NoInputNodesInImportedDecisionRule.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    public boolean accepts(final RuleExtension rule,
+                           final GraphConnectionContext context) {
+        if (context.getTarget().isPresent()) {
+            final Node<? extends View<?>, ? extends Edge> target = context.getTarget().get();
+            final View<?> content = target.getContent();
+            if (Objects.isNull(content)) {
+                return false;
+            }
+            final Object definition = content.getDefinition();
+            return definition.getClass().getName().equals(rule.getId());
```
Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-api/src/main/java/org/kie/workbench/common/dmn/api/rules/NoInputNodesInImportedDecisionRule.java]: Could we please use `Objects.equals` ?

**Comment by jomarko** [comment]
### Manual Review
Passed, thank you @danielzhe

**Comment by danielzhe** [comment]
Changes applied. Thank you, @jomarko

**Pull request merged**